server/dex/dex.go

```diff
@@ -228,6 +233,12 @@ func (dm *DEX) handleDEXConfig(interface{}) (interface{}, error) {
 	return dm.configResp.configEnc, nil
 }
 
+// FeeCoiner describes a type that can check a transaction output, namely a fee
+// payment, for a particular asset.
+type FeeCoiner interface {
+	FeeCoin(coinID []byte) (addr string, val uint64, confs int64, err error)
```

Comment:
Not really returning a `Coin`, but I don't care that much.

Reply:
I didn't like the name either when doing this but I stuck to the common [interface naming guidelines](https://golang.org/doc/effective_go#interface-names) here for some reason.

Rightly, we often veer off that guidance when it makes a better interface name, and I think this is such a case.  Maybe `FeeChecker` instead?  Not sure if I want to rename the actual `FeeCoin` methods too, but those could become `CheckFee` or since the methods are actually performing generic checks of a coin, `CheckCoin` methods for a `CoinChecker` interface?

Reply:
I do think that `CheckCoin` is more descriptive, but I really really don't care that much.